README.md

```diff
@@ -1,10 +1,6 @@
-# Digital Identity IPV UK Passport CRI API
-V1 iteration of the passport cri backend. This is to align with other CRIs
-`di-ipv-cri-uk-passport-api`
+# Digital Identity Passport Credential Issuer
```

Comment:
Could we add a short description here on what the passport CRI does and a bit about how V1 now uses the common lambdas approach.